controllers/auth_config_controller.go

```diff
@@ -594,9 +641,13 @@ func (r *AuthConfigReconciler) translateAuthConfig(ctx context.Context, authConf
 	}
 
 	// denyWith
-	if denyWith := authConfig.Spec.DenyWith; denyWith != nil {
-		translatedAuthConfig.Unauthenticated = buildAuthorinoDenyWithValues(denyWith.Unauthenticated)
-		translatedAuthConfig.Unauthorized = buildAuthorinoDenyWithValues(denyWith.Unauthorized)
+	if authConfig.Spec.Response != nil {
+		if denyWith := authConfig.Spec.Response.Unauthenticated; denyWith != nil {
+			translatedAuthConfig.Unauthenticated = buildAuthorinoDenyWithValues(denyWith)
+		}
+		if denyWith := authConfig.Spec.Response.Unauthorized; denyWith != nil {
+			translatedAuthConfig.Unauthorized = buildAuthorinoDenyWithValues(denyWith)
+		}
```

Comment:
Could probably merge into that other block above where building the success custom responses?

Reply:
Can't _that_ easily, so I'll keep like so for now. Just to keep to diff "cleaner", as this would require a much bigger refactoring to save on `if` statement...